pkg/eval/json_evaluator.model.go

```diff
@@ -1,12 +1,17 @@
 package eval
 
+import (
+	"encoding/json"
+)
+
 type Flags struct {
 	Flags map[string]Flag `json:"flags"`
 }
 
 // we could make this more type-safe with generics when we upgrade to 1.18.
 type Flag struct {
-	State string `json:"state"`
-	DefaultVariant string	`json:"defaultVariant"`
-	Variants map[string]interface{} `json:"variants"`
+	State 					string									`json:"state"`
+	DefaultVariant	string									`json:"defaultVariant"`
+	Variants 				map[string]interface{}	`json:"variants"`
+	Rules 					json.RawMessage					`json:"rules"`
```

Comment:
Used `json.RawMessage` as the type here, since we are just going to convert it to bytes again for use in json-logic evaluation anyway.